settings.gradle

```diff
@@ -169,6 +169,7 @@ include 'data-prepper-plugins:buffer-common'
 //include 'data-prepper-plugins:http-sink'
 //include 'data-prepper-plugins:sns-sink'
 //include 'data-prepper-plugins:prometheus-sink'
+//include 'data-prepper-plugins:personalize-sink'
```

Comment:
I'm ok with un-commenting this out (unless it causes the build to fail).

Reply:
sounds good I'll un-comment this